plugins/dist_table_procedure.h

```diff
@@ -185,10 +185,12 @@ struct DistTableProcedure: public Procedure {
 
     DistTableProcedureConfig procConfig;
 
-    static void persist(const Url & modelFileUrl, DistTableMode mode, 
+    static void persist(const Url & modelFileUrl, DistTableMode mode,
                         const DistTablesMap & distTablesMap);
 
-    static const int DIST_TABLE_PERSIST_VERSION = 2;
```

Comment:
I already tried the `constexpr int DIST_TABLE_PERSIST_VERSION = 2;` version but it still generates a ton of messages.

The enum version works fine though. 👍